src/app/components/navbar/navbar.component.ts

```diff
@@ -15,19 +15,14 @@ import { ChangecolorDirective } from '../../shared/changecolor.directive';
 
 
 export class NavbarComponent {
-  isLoggedIn: WritableSignal<boolean> = signal<boolean>(false)
 
   constructor(private authService: AuthService, private router: Router) {
 
   }
 
-  ngOnInit(): void {
-    this.isLoggedIn = this.authService.getLoggedIn()
-  }
 
   logout(){
      this.authService.logout()
-     this.authService.setLoggedIn(false)
      this.router.navigate(['login'])
```

Comment:
you can consider moving your navigation to your authService.logout if you want your users to be redirected to the login page when logged out

Reply:
currently using the store to handle my logout functionality